Libraries/Image/ImageProps.js

```diff
@@ -151,7 +151,7 @@ export type ImageProps = {|
    *
    * See https://reactnative.dev/docs/image#source
    */
-  source?: ?ImageSource,
+  source: ?ImageSource,
```

Comment:
The `<Image/>` component also has a `src` property, that accepts a regular string. 

If we make the source property required, code that uses the src property will now have to be modified to contain source={undefined}. This degrades the ergonomics for the Image component.